### PR TITLE
#165413081 list bookmarked articles 

### PR DESCRIPTION
**Pull request by Kabalisa**
Description
=======
This feature is about allowing an authenticated user to see articles they bookmarked in their profile.

Type of change
=======
N/A

How has it been tested
=======
- pull this branch
- run ```npm install```
- run ```npm start``` to start our react app
-login into authorshaven
-open your profile.
-if you have bookmarked articles you will see them. if not you will see a message indicating you have none.
-Click on the title or image of the bookmarked article to navigate to it.

Checklist:
=======
N/A

Pivotal tracker story ID
=======
[#165413081](https://www.pivotaltracker.com/story/show/165413081)

Screenshot
=======
Screen Shot 2019-07-29 at 12 50 53

### CODE REVIEW

**Review by jabichris — APPROVED** [review]
Nice work

**Review by Niyitangasam — APPROVED** [review]

**Comment by macphilips** [comment]
Please provide an image of the UI you implemented in the description of this PR

File: src/components/profile/viewProfile.jsx [hunk 1]
```diff
@@ -221,6 +239,57 @@ class ViewProfile extends Component {
                     then type all required information
                   </div>
                 )}
+                <div className='bookmark'>
+                  <h1>Bookmarks</h1>
+                  {isBookmarksFetched === 'done' && Bookmarks.length > 0 ? (
+                    Bookmarks.map(bookmark => {
```
Comment (macphilips) [inline src/components/profile/viewProfile.jsx:245]: Where is `Bookmarks` coming from?

Comment (Kabalisa) [reply src/components/profile/viewProfile.jsx:245]: ```Bookmarks``` is coming from props. if you can check on line 143 of ```viewProfile.jsx```  where the render method starts, I destructure ```bookmarks``` from ```this.props``` then on line 145 I destructure ```Bookmarks``` from ```bookmarks```.

**Comment by Kabalisa** [comment]
> Please provide an image of the UI you implemented in the description of this PR

@AJAkimana .when someone bookmarks again, their bookmark is removed.

**Pull request merged**